packages/model-viewer/src/features/scene-graph/api.ts

```diff
@@ -71,6 +71,16 @@ export declare interface Material {
    * The PBRMetallicRoughness configuration of the material.
    */
   readonly pbrMetallicRoughness: PBRMetallicRoughness;
+
+  /**
+   * Asynchronously loads the underlying material resource if it's currently
+   * unloaded, otherwise the method is a noop.
+   */
+  ensureLoaded(): void;
+  /**
+   * Returns true if the material is loaded and valid.
+   */
+  isValid: boolean;
```

Comment:
Maybe `isLoaded`? What would cause a material to be invalid?

Reply:
If we do some validation step in the future - which always comes with materials, I've just done this dance many times with materials but maybe I'm getting too far ahead.

Reply:
glTF validation is generally a pre-process anyway. If it managed to successfully load in three.js, I think it's plenty valid enough for us.